document/core/exec/runtime.rst

```diff
@@ -34,12 +35,14 @@ or *external references* pointing to an uninterpreted form of :ref:`extern addre
      \I64.\CONST~\i64 \\&&|&
      \F32.\CONST~\f32 \\&&|&
      \F64.\CONST~\f64 \\
+   \production{(vector)} & \vecc &::=&
+     \V128.\CONST~\i128 \\
```

Comment:
is there any documentation we can refer to that clarifies why this is v128.const and not i128.const for declaring an i128? Or is this possibly a bug in the spec?

Reply:
I guess it is a shortcut rather than declaring i128.const and then another vector instruction that produces it? nevertheless is there a plan or explicit plan against i128.const?

Reply:
PS: As for documentation, that is essentially said in https://webassembly.github.io/spec/core/syntax/values.html#vectors